packages/js/product-editor/src/components/more-menu-dropdown/index.tsx

```diff
@@ -21,6 +21,13 @@ export const MoreMenuDropdown = ( {
 	popoverProps,
 	toggleProps,
 	children,
+}: {
+	as?: React.ElementType;
+	className?: string;
+	label?: string;
+	popoverProps?: Record< string, unknown > & { className?: string };
+	toggleProps?: Record< string, unknown >;
+	children: ( onClose: () => void ) => React.ReactNode;
```

Comment:
```suggestion
	children: ( props: { onClose: () => void } ) => React.ReactNode;
```
Should be the above I believe?

Reply:
Actually on second look, scratch my suggestion, it looks like your type is correct with how it is used down below.

In that case we should just fix the type in the `more-menu.tsx` file.